docker/scripts/yarn-setup.js

```diff
@@ -1,6 +1,10 @@
 const { spawnSync: baseSpawnSync } = require("child_process");
 const { basename, dirname, resolve } = require("path");
 const platform = require("os").platform();
+const { safeRandomString } = require("../../scripts/lib/random");
+const fsp = require("fs").promises;
+
+const DOTENV_PATH = `${__dirname}/../../.env`;
```

Comment:
or we should just use `docker.env` here instead of `.env`

why does it need to be in `.env`? it's only relevant for docker right?

Reply:
Basically: `docker.env` is committed, `.env` is not, and we need `ROOT_DATABASE_URL` to contain the password we've generated for `POSTGRES_PASSWORD` so we cannot write it to `docker.env` otherwise it'd be tracked under git. So instead I just have Postgres source the `.env` file.

Oh... I see what you mean - this breaks dual Docker/local dev. I don't think that's an issue, TBH - we've told users to pick one and stick to it in the README so it should be fine? Maybe we should add a `docker/.env` file that contains these secrets and source both, though. It looks like (though the docs don't say) `env_file` could take an array. I'm going to give this a try.